docs/scripts/formattedTSDemos.js

```diff
@@ -1,55 +1,137 @@
+/* eslint-disable no-console */
+
 /**
- * Transpiles and formats TS demos.
+ * Transpiles TypeScript demos to formatted JavaScript.
  * Can be used to verify that JS and TS demos are equivalent. No introduced change
  * would indicate equivalence.
  */
-const childProcess = require('child_process');
+
+/**
+ * List of demos to ignore when transpiling
+ * Example: "app-bar/BottomAppBar.tsx"
+ */
+const ignoreList = [];
```

Comment:
Why would we ignore a demo?

Reply:
Because I received quite some push back when proposing TS demos so I had to go out of my way to make sure they won't block builds.